namer/consul/src/test/scala/io/buoyant/namer/consul/ConsulNamerTest.scala

```diff
@@ -471,7 +471,7 @@ class ConsulNamerTest extends FunSuite with Awaits {
     ))
   }
 
```

Comment:
I think we should add a test for the case when a DC is created and destroyed.  This would use a TestApi that returns a 500 response for the first request (representing that the DC does not exist yet), a valid 200 response for the second request (representing that the DC has been created), and then a 500 response again for the third request (representing that the DC has been deleted). 

We should check that the state is Neg after the first request, Bound after the second, and Neg again after the third.